### ISSUE TITLE
error/warning in create script dialog (or all dialogs?) should be read.

### ISSUE THREAD

**Issue by manodrum** [earsketch/earsketch-webclient#1059]
For example, trying to create a script with a single character name shows an error message which is not read by the screenreader, so screenreader users don't know why they can't proceed.

This may be an issue globally with all dialogs using our template.
